packages/runtime/container-runtime/src/summary/summarizerNode/summarizerNode.ts

```diff
@@ -166,7 +166,7 @@ export class SummarizerNode implements IRootSummarizerNode {
 		telemetryContext?: ITelemetryContext,
 	): Promise<ISummarizeResult> {
 		// Try to reuse the tree if unchanged
-		if (this.canReuseHandle && !fullTree && !this.hasChanged()) {
+		if (this.canReuseHandle && !fullTree && !this.hasChanged() && trackState) {
```

Comment:
What about line 190?  Want to update this function to only take 1 bool?

Reply:
I tried removing `trackState` before but it's more trouble than worth. I just added logic to handle trackState separtely here.